mypy/server/trigger.py

```diff
@@ -1,5 +1,17 @@
 """AST triggers that are used for fine-grained dependency handling."""
 
+# Used as a suffix for triggers to handle "from m import *" dependencies (see also
+# make_wildcard_trigger)
+WILDCARD_TAG = '[wildcard]'
+
 
 def make_trigger(name: str) -> str:
     return '<%s>' % name
+
+
+def make_wildcard_trigger(module: str) -> str:
```

Comment:
Could you add some comments about the semantic difference between module triggers and module wildcard triggers?

Reply:
Will add.